governance-exercises/VoteWizard_poll_exercise_01/README.md

```diff
@@ -0,0 +1,82 @@
+
+3. **Consideration of Governance Cycle (MIP16 vs MIP51):** As part of this process, I would evaluate whether the proposal should be integrated into the Weekly Governance Cycle (as per MIP16) or the Monthly Governance Cycle (as per MIP51). Given the nature and urgency of this particular proposal, I would opt for inclusion in the Weekly Governance Cycle beginning on Monday, July 24th. This decision ensures timely action while recognizing that not every decision requires awaiting the Monthly Governance Cycle, which is primarily used for introducing MIP subproposals into the Maker Protocol.
+
+4. **Poll Template Identification:** Next, I would identify the most suitable template to construct this poll. In this situation, there are two relevant templates: 'Adjust $protocol_name D3M Parameters - $date_MONTH_DD,_YYYY.md' and 'Onboard $token to Spark Protocol - $date_MONTH_DD,_YYYY.md'.
+
+5. **Poll Separation Decision:** While combining both templates into a singular poll might seem efficient, I argue that the onboarding of a new token to the Spark Protocol and the adjustment of Spark Protocol parameters are distinct actions. It's crucial to provide voters the opportunity to express their opinions on these separate matters.
+
+	- As a result, I propose two separate polls:
+		- Poll 1: Adjust Spork Protocol D3M Parameters and DAI Interest Rate Strategy - July 24, 2023
+			- Increase of the Spork D3M Debt Ceiling from 50 million to 100 million DAI.
+			- Increase of the Spork D3M Target Available Debt from 5 million to 15 million.
+			- Adjust Spork Protocol DAI Interest Rate Strategy to increase the DAI deposit rate from 2% to 2.5%.
+			- Increase the Ceiling Increase Cooldown from 28,800 seconds (8 hours) to 86,400 seconds (24 hours)
+		- Poll 2: Onboard ABC to Spork Protocol - July 24, 2023
+			- Onboard the ABC to Spork with the parameters below:
+			- [onboarding parameters for the token ABC]
+
+6. **Forum Communication:**  Afterward, I would publish a forum post detailing the decision to submit these two independent polls. This action will ensure transparency and keep the community informed.
+
+7. **Draft the Governance Polls:** The next step would be to draft the governance polls. The poll would include a brief overview of the proposal, clear voting options, and links to relevant resources for voters to make an informed decision. It's important to make sure that the text is impartial and without bias toward specific or general outcomes.
```

Comment:
No need to go to quite so much detail in writing these VoteWizard. 'draft the governance polls' would have been fine :)